packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -1077,38 +1075,10 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
         shouldPersistHeaders as boolean,
         (result: FetcherResult) => {
           if (queryID === this._editorQueryID) {
-            if (Array.isArray(result)) {
-              // for `IncrementalDelivery`
-              // https://github.com/graphql/graphql-over-http/blob/master/rfcs/IncrementalDelivery.md
-              // TODO: typescript types
-              const response = result.reduce((result, increment) => {
-                if (increment.errors) {
-                  result.errors = [
-                    ...(result?.errors || []),
-                    ...increment?.errors,
-                  ];
-                }
-                if (increment.path) {
-                  const [path, index] = increment.path;
-                  const data = result?.data[path] || [];
-                  // place them at the exact index. this matters a lot
-                  data[index] = increment.data;
-                  result.data = { ...result?.data, [path]: data };
-                } else {
-                  result.data = { ...result?.data, ...increment.data };
-                }
-                return result;
-              }, totalResponse);
-              this.setState({
-                isWaitingForResponse: false,
-                response: GraphiQL.formatResult(response),
-              });
-            } else {
-              this.setState({
-                isWaitingForResponse: false,
-                response: GraphiQL.formatResult(result),
-              });
-            }
+            this.setState({
```

Comment:
without aggregating `totalResponse` this won't work, I don't thinK?

Reply:
ya. haha correcting.